ui/components/Link.tsx

```diff
@@ -12,7 +13,19 @@ type Props = {
   href?: any;
   newTab?: boolean;
   textProps?: TextProps;
+  icon?: JSX.Element;
   onClick?: (ev: any) => void;
+  onMouseEnter?: React.EventHandler<React.SyntheticEvent>;
+  onMouseLeave?: React.EventHandler<React.SyntheticEvent>;
+};
+
+const SpacedIcon = ({ icon }: { icon: JSX.Element }) => {
+  return (
+    <>
+      {icon}
+      <Spacer padding="xxs" />
+    </>
+  );
 };
```

Comment:
For similar components which only return a value and don't assign any local vars/don't use local functions, we could use a simpler return format for arrow functions:

```
const SpacedIcon = ({ icon }: { icon: JSX.Element }) => (
  <>
    {icon}
    <Spacer padding="xxs" />
  </>
);
```